dispatch/apps/frontend/templatetags/widgets.py

```diff
@@ -11,6 +12,10 @@ def zone(zone_id):
     try:
         zone = ThemeManager.Zones.get(zone_id)
     except ZoneNotFound:
-        return None
+        return ''
+
+    widget = zone.widget
+    if widget is not None:
```

Comment:
I think I'd prefer this (I tend to avoid unnecessary variable assignment, but I'm not sure what's more readable):

```python
if zone.widget:
  return zone.widget.render()